src/core/error_mod.F90

```diff
@@ -96,6 +99,8 @@ END SUBROUTINE CC_Error
    !!
    !! This subroutine prints a warning message and sets RC to CC_SUCCESS.
    !!
+   !! \ingroup Core_Modules
```

Comment:
Is it not enough for the module to belong to Core_Modules group?

Reply:
This adds the function itself to the group. If you don’t it won’t see the function under the grouping just the file